thrust/thrust/system/cuda/detail/transform_reduce.h

```diff
@@ -67,7 +67,7 @@ namespace detail
 {
 
 template <typename Derived, typename InputIt, typename Size, typename UnaryOp, typename T, typename BinaryOp>
-THRUST_RUNTIME_FUNCTION T transform_reduce_n_impl(
```

Comment:
That seems like a mistake

Reply:
Whoops. Pressed space and didn't realize it.